app/controllers/make_a_demo_payment/go_to_payment_controller.js

```diff
@@ -38,7 +40,8 @@ module.exports = (req, res) => {
       lodash.unset(req, 'session.pageData.makeADemoPayment')
       res.redirect(product.links.pay.href)
     })
-    .catch(() => {
+    .catch((err) => {
+      logger.error(`[requestId=${req.correlationId}] Making a demo payment failed - ${err.message}`)
```

Comment:
Have you tested this (in integration). I guess we have a slight difference from `publicauth` and `products` the way we return error messages. products doesn't send `.message`
How about 
```
if(err.message) {
logger.error(`${err.message}`)
} else {
logger.error(`${JSON.stringify(err}`)
}
```

Reply:
or just `JSON.stringify` for both ?

Reply:
Tests shows that always an Error object is returned, which always have a message